src/calculateRentalCost.js

```diff
@@ -29,6 +29,21 @@
 
 function calculateRentalCost(days) {
   // write code here
+  const price = 40;
+
+  if (days <= 0) {
+    return 0;
+  }
+
+  if (days < 3) {
+    return days * price;
+  }
+
+  if (days < 7) {
+    return days * price - 20;
+  }
+
+  return days * price - 50;
```

Comment:
Move this magic numbers '20' and '50' to variables

Reply:
i jak mi to teraz zrobić